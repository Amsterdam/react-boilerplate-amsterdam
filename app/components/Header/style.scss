@import '~stijl/dist/scss/ams-colorpalette';

.header-component {
  .mainlogo {
    text-decoration: none;
  }
  .sitelogo {
    display: inline-block;
  }
}
