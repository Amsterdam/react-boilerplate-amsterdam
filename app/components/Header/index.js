import React from 'react';
import './style.scss';

import LogoSvg from '../../../node_modules/stijl/dist/images/logos/andreas.svg';
import LogoPng from '../../../node_modules/stijl/dist/images/logos/andreas.png';
import LogoPrint from '../../../node_modules/stijl/dist/images/logos/andreas-print.png';

class Header extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    return (
      <div className="header-component">
<<<<<<< HEAD
        <div className="row">
          <div className="col-6">
            <h1 className="sitelogo">
              <a className="mainlogo" href="https://www.amsterdam.nl">
                <span className="logoset">
                  <img src={LogoSvg} className="screen-logo" alt="Gemeente Amsterdam" />
                  <img src={LogoPng} className="alt-logo" alt="Gemeente Amsterdam" />
                  <img src={LogoPrint} className="print-logo" alt="Gemeente Amsterdam" />
                </span>
                <span className="logotexts">
                  <span className="logotext red">Gemeente</span>
                  <span className="logotext red">Amsterdam</span>
                </span>
              </a>
            </h1>
          </div>
          <div className="col-6 type-nav-secundair ">
            <nav>
              <ul className="links horizontal right ">
                <li>
                </li>
              </ul>
            </nav>
          </div>
        </div>
=======
        <h1>Signalen</h1>
        <Link to="/">
          <FormattedMessage {...messages.home} />
        </Link>
        <Link to="/features">
          <FormattedMessage {...messages.features} />
        </Link>
        <Link to="/incident">
          <FormattedMessage {...messages.incident} />
        </Link>
        <Link to="/admin/incidents">
          <FormattedMessage {...messages.admin} />
        </Link>
>>>>>>> add: navigation to incident detail
      </div>
    );
  }
}

export default Header;
