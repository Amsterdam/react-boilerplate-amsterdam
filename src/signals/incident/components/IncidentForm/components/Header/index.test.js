import React from 'react';
import { shallow } from 'enzyme';

import Header from './index';

describe('Form component <Header />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it('should render correctly', () => {
    wrapper.setProps({
      meta: {
        label: 'Header'
      }
    });

    expect(wrapper).toMatchSnapshot();
  });
});
