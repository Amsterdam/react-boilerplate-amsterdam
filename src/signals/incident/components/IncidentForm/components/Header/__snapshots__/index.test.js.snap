// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component <Title /> should render correctly 1`] = `
<div
  className=" col-12 title"
>
  <div
    className="title__label label"
  >
    Title
  </div>
  <div
    className="title__subtitle"
  />
</div>
`;
